src/app/app.module.ts

```diff
@@ -227,6 +228,8 @@ export function translationServiceFactory(translate: TranslateService, injector:
     forwardRef(() => ArlasBookmarkService),
     forwardRef(() => ArlasConfigService),
     forwardRef(() => ArlasCollaborativesearchService),
+    forwardRef(() => ArlasStartupService),
+    forwardRef(() => ArlasConfigurationDescriptor),
     forwardRef(() => ArlasColorGeneratorLoader),
     forwardRef(() => ArlasExtendService),
     forwardRef(() => ArlasStartupService),
```

Comment:
`forwardRef(() => ArlasStartupService)` is added twice

Reply:
Done